topics.md

```diff
@@ -16,36 +16,49 @@ This is meant to be a guide -- don't let it overwhelm you. Give us what you've g
 * Requirements: [A/V, room setup, or any other needs]
 * Code of Conduct: I have read, understand, and agree to follow the Tech Talk DC Code of Conduct.
 
-## Get into Docker Workshop Series, Part I: Getting Started with Docker 
-
-* Session Title: Getting Started with Docker Workshop
-* Preferred Timeframe: A March evening 
-* Event Type: 2 Hour Workshop - 6-8, w/happy hour after
-* Summary: We will take an existing simple app and Dockerize it. I'll run through the concepts of Docker for ~20 minutes as this is for getting started with Docker, then it will be Dockerizing an app and then using Compose to build and run it.
-* Presenter Name(s): Wyn Van Devnater
-* Presenter E-mail(s): wyn.vandevanter@execlla.com
-* Presenter GitHub Account: https://github.com/wyntuition
-* Presenter Blog: http://www.excella.com/insights
-* Audience Skill Level: Beginners
-* Requirements: Table at the front I can sit at while projecting working on my laptop, workshop-style 4 person-per-table setup, I think both sides of the ATX will be needed
-* Code of Conduct: I have read, understand, and agree to follow the Tech Talk DC Code of Conduct.
-
-## Get into Docker Workshop Series, Part 2: Getting Started with Docker Deployments
-
-* Session Title: Getting Started With Docker Deployments
-* Preferred Timeframe: An April evening 
-* Event Type: 2 Hour Workshop - 6-8, w/happy hour after
-* Summary: We will take an existing simple app that is running in Docker, and take the steps to deploy it. We'll use AWS ECS since it's already set up and we can focus on the app. 
-* Presenter Name(s): Wyn Van Devnater
-* Presenter E-mail(s): wyn.vandevanter@execlla.com
-* Presenter GitHub Account: https://github.com/wyntuition
-* Presenter Blog: http://www.excella.com/insights
-* Audience Skill Level: Beginners
-* Requirements: Table at the front I can sit at while projecting working on my laptop, workshop-style 4 person-per-table setup, I think both sides of the ATX will be needed
-* Code of Conduct: I have read, understand, and agree to follow the Tech Talk DC Code of Conduct.
-
-
 ## Accepted Topics
 
 ------------------------------
  
+### Queeries
+  
+  * Session Title: Queeries
```

Comment:
Suggestion (feel free to ignore): Maybe we can revise the title to include something about Tech?  e.g. "Tech Queeries" "Queeries in/on Tech."

Reply:
I'd be open to adding to the name. Let me think about it tonight